phrosty/imagesubtraction.py

```diff
@@ -517,7 +551,8 @@ def swarp_coadd_img(imgpath_list,refpath,out_name,out_path=output_files_rootdir,
     :rtype: str
     """
     cd = PY_SWarp.Mk_ConfigDict(GAIN_DEFAULT=1., SATLEV_DEFAULT=100000., 
-                                RESAMPLING_TYPE='BILINEAR', WEIGHT_TYPE='MAP_WEIGHT', **kwargs)
+                                RESAMPLING_TYPE='BILINEAR', WEIGHT_TYPE='NONE', 
```

Comment:
This looks like WEIGHT_TYPE="NONE" for all coadditions, not just PSF coaddition?  Is that the desired behavior?

Reply:
Yes, I don't have weight maps anyway.

Reply:
FWIW, in principle you do of a weight map.  The variance image is just the image, because gain=1, and we haven't done any processing.  But because observing conditions are really pretty constant, it doesn't really matter here.